common-lib/test/cmr/common/test/util.clj

```diff
@@ -773,3 +773,15 @@
 
     "Set of numbers"
     #{1 7 52 3} 52))
+
```

Comment:
Nice!  And thanks :-) 

(this gets us one step closer to the goal of putting as much testing as possible in unit tests, and working to keep the sys-int tests as tightly focused on integration-tests-only as possible :-) )

Reply:
Thanks for suggesting it.